firebaseai/FirebaseAIExample/ContentView.swift

```diff
@@ -57,10 +57,19 @@ struct ContentView: View {
             Label("Multi-modal", systemImage: "doc.richtext")
           }
           NavigationLink {
-            ConversationScreen(firebaseService: firebaseService)
+            ConversationScreen(firebaseService: firebaseService, title: "Chat")
           } label: {
             Label("Chat", systemImage: "ellipsis.message.fill")
           }
+          NavigationLink {
+            ConversationScreen(
+              firebaseService: firebaseService,
+              title: "Grounding",
+              searchGroundingEnabled: true
+            )
+          } label: {
+            Label("Grounding", systemImage: "magnifyingglass")
```

Comment:
Would you mind if we changed this to "Grounding with Google Search"?

Note: I think it'll be too wide to put "Grounding with Google Search" as the title above (line 67) so I'd just leave that one as is.

Reply:
Done